__tests__/app.test.js

```diff
@@ -86,4 +87,70 @@ describe('testing GET /api/articles/:article_id', () => {
             expect(data._body.msg).toBe("Invalid input: 400 Bad Request");
         })
     });
-});
\ No newline at end of file
+});
+
+describe('testing GET /api/articles', () => {
+    test('should respond with status 200 and an articles array of all article objects each with the required properties', () => {
+
+        // make sure body property is not included in final list of objects
+
+        return request(app).get("/api/articles")
+        .expect(200)
+        .then(response => {
+            expect(response._body.articles.length > 0).toEqual(true); // check for empty arr
+            response._body.articles.forEach((article => {
+                expect(typeof article.author).toBe("string")
+                expect(typeof article.title).toBe("string")
+                expect(typeof article.article_id).toBe("number")
+                expect(typeof article.topic).toBe("string")
+                expect(typeof article.created_at).toBe("string")
+                expect(typeof article.votes).toBe("number")
+                expect(typeof article.article_img_url).toBe("string")
+                // will test for comment_count in next test
+            }))
+        })
+    });
+    test('should be sorted by date in descending order', () => {
+        // do a loop, check in each one the date is not before each other
+        return request(app).get("/api/articles")
+        .then(data => {
+            let resultArr = data._body.articles; // putting in variable for convenience
+            expect(resultArr.length > 0).toEqual(true); // check for empty arr
+            for (let i = 0; i < resultArr.length - 2; i++) { // loop until 2nd last
+                // convert strings to date then to numbers to compare
+                expect(new Date(resultArr[i].created_at).getTime()).toBeGreaterThanOrEqual(new Date(resultArr[i + 1].created_at).getTime())
+            }
+        })
+    });
```

Comment:
This is fine, but the npm package `jest-sorted` might make this alot easier for you!